### DISCUSSION TITLE
Can I use this in my own application?

### DISCUSSION THREAD

**Discussion by itjustcrashed** [IdreesInc/Monocraft#143] [General]
I am developing my very own Minecraft launcher to make launching the game on Mac not the most terrifying thing you've ever seen.
Can I use this font in it?

Writing in Xcode never felt more minecraft-y

**Comment by Ciubix8513** [comment]
It's OFL, so you can basically do whatever with the font, [faq for using ofl fonts](https://openfontlicense.org/how-to-use-ofl-fonts/)

**Comment by itjustcrashed** [comment]
Alrighty!
